haskell-src-transformations.cabal

```
@@ -64,8 +98,9 @@ test-suite haskell-src-transformations-unit-tests
   main-is:            Spec.hs
   ghc-options:        -main-is Spec
   other-modules:      HST.CoreAlgorithmTests
+                    , HST.Effect.CancelTests
   build-depends:      base
```

Shouldn't a version restriction be added here as well?
```suggestion
  build-depends:      base              >= 4.12 && < 4.13
```

---

See 9000bb5d993b702a41bb40e576b18ebf11a7c5dd.